deploy/Chart/templates/controller/webhook.yaml

```diff
@@ -32,7 +32,7 @@ webhooks:
 - admissionReviewVersions:
   - v1
   clientConfig:
-    caBundle: {{ b64enc $ca.Cert }}
+    caBundle: {{ if $existingValidatingWebhook }}{{ index $existingValidatingWebhook "webhooks" 0 "clientConfig" "caBundle" }}{{ else }}{{ b64enc $ca.Cert }}{{ end }}
```

Comment:
@ytimocin What if we can fetch the cert from secret instead of fetching from webhook resource? Secret resource is source of truth, so it is better to fetch from secret resource. bitnami hosting chart is using this way. Please see the below example.

https://github.com/bitnami/charts/blob/882140cfc5318420fdf563b77201f7ee5f67ca8f/bitnami/rabbitmq-cluster-operator/templates/messaging-topology-operator/validating-webhook-configuration.yaml#L51

If you think it is better approach, please update this way in webhook controller and ucp.